src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return String.valueOf(Color.values()[index]).toLowerCase();
+
+    }
```

Comment:
remove empty line
let's make `Random` class level field so we don't create objects each time we call `getRandomColor()`
return `Color` not `String`